tests/FixtureBuilder/Payload.php

```diff
@@ -0,0 +1,169 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tuf\Tests\FixtureBuilder;
+
+use Tuf\CanonicalJsonTrait;
+
+/**
+ * A payload is a builder for a server-side TUF metadata file.
+ *
+ * A payload represents a single role in the TUF repository. This could be one
+ * of the four top-level roles (root, timestamp, snapshot, and targets), or it
+ * could be a delegated targets role at any level of nesting.
+ *
+ * Every payload has relationships to other payloads. There are two kinds
+ * of relationships:
+ *
+ * - A "parent" relationship is for a payload that enumerates other payloads.
+ *   For example, a snapshot has a parent relationship to all targets roles,
+ *   regardless of nesting level. The parent may needs to be updated when one
```

Comment:
```suggestion
 *   regardless of nesting level. The parent may need to be updated when one
```
Not sure if should be `may need` or just `needs`